test/metabase/util_test.cljc

```diff
@@ -508,3 +509,21 @@
        (let [expected (reduce + (u/map-all (fnil + 0 0 0) xs ys zs))
              sum (+ (reduce + 0 xs) (reduce + 0 ys) (reduce + 0 zs))]
          (= expected sum)))))
+
+#?(:clj
+   (deftest ^:parallel case-enum-test
+     (testing "case does not work"
+       (is (not (case Month/MAY
+                  Month/APRIL false
+                  Month/MAY   true
+                  false))))
```

Comment:
nit: I'd prefer this test with distinct values for all three branches so it's clearer.
```suggestion
       (is (= 3 (case Month/MAY
                  Month/APRIL 1
                  Month/MAY   2
                  3))))
```